client/src/main/java/com/alibaba/nacos/client/utils/ParamUtil.java

```diff
@@ -164,12 +164,7 @@ public static String parseNamespace(Properties properties) {
                         String.valueOf(Constants.DEFAULT_USE_CLOUD_NAMESPACE_PARSING)));
         
         if (Boolean.parseBoolean(isUseCloudNamespaceParsing)) {
-            namespaceTmp = TemplateUtils.stringBlankAndThenExecute(namespaceTmp, new Callable<String>() {
-                @Override
-                public String call() {
-                    return TenantUtil.getUserTenantForAcm();
-                }
-            });
+            namespaceTmp = TenantUtil.getUserTenantForAcm();
```

Comment:
Why change this code?

Reply:
line 160  String namespaceTmp = null ,  namespace is null here,  stringBlankAndThenExecute is  redundant

Reply:
line 160 String namespaceTmp = null , namespace is null here, stringBlankAndThenExecute is redundant